cmd/analyzer/main.go

```diff
@@ -29,6 +30,7 @@ var (
 	outputViolations  = flag.String("output-file", "output.yaml", "filepath to to store rule violations")
 	errorOnViolations = flag.Bool("error-on-violation", false, "exit with 3 if any violation are found will also print violations to console")
 	labelSelector     = flag.String("label-selector", "", "an expression to select rules based on labels")
+	depLabelSelector  = flag.String("dep-label-selector", "", "an expression to select dependencies based on labels. This will filter out the violations from these dependencies as well these dependencies when matching dependency conditions.")
```

Comment:
this is super bikesheddy so feel free to disregard, but is this more of a `selector` or a `filter`? I feel like the word `filter` might be more clear based on the description?

Reply:
You can filter in, and you can filter out and you can select for inclusion and exclusion. The hard part for me is which is better.